STYLE_GUIDE.md

```diff
@@ -0,0 +1,443 @@
+```pony
+actor Main
+  new create(env: Env) =>
+    env.out.print("Hello, world!")
+```
+
+### Blank Lines
+
+- One blank line is placed between top-level declarations such as classes, primitives, actors, etc.
+- There is no blank lines between the class declaration and its first member or documentation.
+- Fields have no blank lines between them.
+- Functions are separated from other members by one line.
+
+  ```pony
+  // OK
+  class Foo
+  let bar: Bar
+  let baz: Baz
+
+  fun foo() =>
+  // ...
+
+  fun bar() =>
+  // ...
+
+  // OK
+  class Foo
+  """
+  Some documentation
+  """
+  let bar: Bar
+
+  fun foo() =>
+  // ...
+
+  // OK
+  class Foo
+  fun foo() =>
+  // ...
+
+  // Not OK
+  class Foo
+
+  let bar: Bar
+  let baz: Baz
+  fun foo() =>
+  // ...
+  ```
+
+- Blank lines between __one-line__ functions or primitives may be omitted if they are related.
+
+  ```pony
+  primitive Red    fun apply(): U32 => 0xFF0000FF
+  primitive Green  fun apply(): U32 => 0x00FF00FF
+  primitive Blue   fun apply(): U32 => 0x0000FFFF
+
+  type Color is (Red | Green | Blue)
+  ```
+
+### Whitespace
+
+- Use spaces around binary operators and after commas, colons, and semicolons. Do not put spaces around brackets or parentheses.
+
+  ```pony
+  let x: USize = 3
+
+  x = x + (5 * x)
+
+  (let a, let b) = ("a", "b")
+
+  let ns = [as USize: 1; 2; 3]
+  ```
+
+- The `not` operator is surrounded by spaces, but the `-` operator is not followed by a space.
+
+  ```pony
+  if not x then -a end
+  ```
+
+- Lambda expressions follow the rules above except that a space is placed before the closing brace.
+
+  ```pony
+  // OK
+  {(a: USize, b: USize): USize => a + b }
+
+  // Not OK
+  {(a: USize, b: USize): USize => a + b}
+
+  // Not OK
+  { (a: USize, b: USize): USize => a + b }
+  ```
+
+- Normal rules apply to Lambda types. 
+
+  ```pony
+  // OK
+  type foo is {(USize, USize): USize} box
+
+  // Not OK
+  type foo is {(USize, USize): USize } box
+
+  // Not OK
+  type foo is { (USize, USize): USize } box
+  ```
+
+### Type Aliases
+
+Align long union and intersection types with each type on a separate line starting with the `(`, `|`, or `&` symbols. In this case, every symbol must be separated by a single space or a newline.
+
+```pony
+// OK
+type Signed is (I8 | I16 | I32 | I64 | I128)
+
+// OK
+type Signed is
+  ( I8
+  | I16
+  | I32
+  | I64
+  | I128 )
+
+// OK
+type Signed is
+  ( I8
+  | I16
+  | I32
+  | I64
+  | I128
+  )
+
+// Not OK
+type Signed is ( I8
+              | I16
+              | I32
+              | I64
+              | I128 )
+
+// Not OK
+type Signed is
+  (I8
+  |I16
+  |I32
+  |I64
+  |I128)
+```
+
+### Array Literals
+
+Multiline arrays generally have a newline between each entry, though logical exceptions exist (such as tables).
+
+```pony
+// OK
+let ns = [as USize: 1; 2; 3]
+
+// OK
+let strs = ["a"; "b"; "c"]
+
+// OK
+let ns =
+  [ as USize:
+    1
+    2
+    3 ]
+
+// OK
+let ns =
+  [ as USize:
+    1
+    2
+    3
+  ]
```

Comment:
Sylvan likes the opposite of Joe.

Sean doesn't care that much. Sean is good with whatever. 
I suggest rock, paper, scissors between Theo and Joe.